server/database/db.js

```diff
@@ -3,7 +3,8 @@ const mysql = require('mysql');
 const connection = mysql.createConnection({
```

Comment:
May be worth breaking out the credentials to a file that is .gitignore-d in the future but not critical